lib/netext/tls.go

```diff
@@ -0,0 +1,97 @@
+package netext
+
+import (
+	"crypto/tls"
+
+	"github.com/loadimpact/k6/lib"
+	"golang.org/x/crypto/ocsp"
+)
+
+const (
+	OCSP_STATUS_GOOD                   = "good"
```

Comment:
Why did you move those constants here - to avoid some import cycle? 

In any case, I like that they're somewhat encapsulated this way, but the original constants are [still present](https://github.com/loadimpact/k6/blob/4fadcb75cfbeeaa48fc352e6e96c6a9d7c0ea5cb/js/modules/k6/http/http.go#L35-L62) in the `http.go` file, which is a recipe for headaches in the future if we want to add something to them...